ranking.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: ranking
+title: CargoRocket Mapathon
+subpage-title: Mapathon
+description: Hier geht es heiß her! Welche Gemeinde wird digital und sorgt für die beste Datengrundlage für Lastenradrouting?! Klicke auf deine Gemeinde um herauszufinden wie du Daten beitragen kannst. Gewinner gibt es in den Kategorien "Meiste neuen Daten" und "Datenqualität".
```

Comment:
Als "neue Daten" werden alle Poller, Umlaufgitter, Bordsteine und Radwege gezählt, die seit dem 17. April 2021 neu in OSM hinzugekommen sind. 
Unter "Datenqualität" wird die Vollständigkeit der Informationen zu Radwegen und Barrieren in Prozent angegeben. Hierzu gehören die Oberfläche und Breite des Radwegs, die Breite bei Pollern und die Höhe bei Bordsteinen.